### ISSUE TITLE
[a11y] 9.2.4.3 Conclusive sequence for keyboard operation

### ISSUE THREAD

**Issue by tschiebel** [owncloud/web#10733]
audit: https://infinite.owncloud.com/s/PnaAukHGGsLnrMM

https://bitvtest.de/pruefschritt/bitv-20-web/bitv-20-web-9-2-4-3-schluessige-reihenfolge-bei-der-tastaturbedienung

### Personal

- [ ] If you open and exit menus or user-defined tooltips with the keyboard
they remain open. This means that visually hidden elements often retain
many cases receive the keyboard focus. To avoid this, the elements should
elements are best closed automatically when they are exited.
are left.
![image](https://github.com/owncloud/web/assets/132444655/cb655869-0391-4a51-a09c-b80b18c959ce)

- [ ] In the mobile view, when the "Details" area (e.g. "Shares") is open,
visually hidden elements in the background can be accessed by swiping backwards with the
the keyboard. This should be avoided.
![image](https://github.com/owncloud/web/assets/132444655/8d336e9c-d74f-484e-aef6-b37ac1aca029)

### Shares, Spaces

- [ ] See comments on Personal
